doc/contribute/project_roles.rst

```diff
@@ -0,0 +1,281 @@
+* Responsibility to coordinate code merges with maintainers.
+* Responsibility to merge all contributions regardless of their
+  origin and area if they have been approved by the respective
+  maintainers and follow the merge criteria of a change.
+* Responsibility to keep the Zephyr code base in a working and passing state
+  (as per CI)
+
+
+Release Activity
+################
+
+.. figure:: img/img_release_activity.jpg
+     :width: 663px
+     :align: center
+     :alt: Release Activity
+
+
+MAINTAINERS File
+################
+
+Generic guidelines for deciding and filling in the Maintainers' list
+
+* The maintainers file shall coexist with the codeowner file.
+  Maintainers file is used to set assignees, codeowner file is used for
+  more granularity and to add reviewers down to the file level.
+  Codeowner entries do not signify maintainership.
+* We should keep the granularity of code maintainership at a manageable level
+* We should be looking for maintainers for areas of code that are orphaned (i.e. without an explicit maintainer)
+
+  * Un-maintained areas should be indicated clearly in the Maintainers file
+
+* All submitted pull-requests should have an assignee
+* We Introduce a maintainers' hierarchy to address the above point
+
+  * Parent-area maintainer should be acting as default substitute/fallback assignee for un-maintained sub-areas
+  * Area maintainer gets precedence over parent-area maintainer
+
+* Pull-requests may be re-assigned if this is needed or more appropriate
+
+  * Re-assigned by original assignee (see “Assignee” slide)
+
+* Updates to the maintainers’ file should be in a standalone PRs
+* The Maintainer’s file itself shall have a maintainer
+* Architectures, core components, sub-systems, samples, tests
+
+  * Each area shall have an explicit maintainer
+
+* Boards (incl relevant samples, tests), SoCs (incl DTS)
+  * May have a maintainer, shall have a higher-level platform maintainer
+* Drivers
+
+  * Shall have a driver-area (and API) maintainer
+  * Could have individual driver implementation maintainers but preferably collaborator/contributors
+  * In the above case, platform-specific PRs may be re-assigned to respective collaborator/contributor of driver implementation
+
+
+Merge Criteria
+##############
+
+* All continuous integration checks have passed
+
+  * Codeowners
+  * Device Tree
+  * Documentation
+  * Gitlint
+  * Identity/Emails
+  * Kconfig
+  * License
+  * Checkpatch (Coding Style)
+  * Pylint
+  * Sanitycheck + Other Unit tests
+  * Simulated Bluetooth Tests
+
+* Planned
+
+  * Footprint
+  * Code coverage
+  * Coding Guidelines
+  * Static Analysis (Coverity)
+  * Documentation coverage (APIs)
+
+* PR template with checklist
+
+* Minimal of 2 approvals
+
+  * A collaborator from the same subsystem.
+  * Alternately another maintainer of another subsystem
+  * Approval by the assignee
+
+* A minimum review period of 2 days, 4 hours for trivial changes.
```

Comment:
Would you consider linking to https://docs.zephyrproject.org/latest/development_process/dev_env_and_tools.html#give-reviewers-time-to-review-before-code-merge ?

Reply:
I think yes, this is not a problem. Does not modify the content. Will do.